### PR TITLE
Minor Bindings Tweaks

### PR DESCRIPTION
**Pull request by TheBlueMatt**

### CODE REVIEW

File: lightning/src/routing/scoring.rs [hunk 1]
```diff
@@ -168,6 +168,14 @@ impl<'a, T: Score + 'a> LockableScore<'a> for MultiThreadedLockableScore<T> {
 	}
 }
 
+#[cfg(c_bindings)]
+impl<T: Score> MultiThreadedLockableScore<T> {
+	/// Creates a new MultiThreadedLockableScore given an underlying [`Score`].
```
Comment (jkczyz) [inline lightning/src/routing/scoring.rs:173]: nit: ticks around `MultiThreadedLockableScore`

**Review by jkczyz — APPROVED** [review]

**Comment by TheBlueMatt** [comment]
Pushed an extra commit that addresses https://github.com/lightningdevkit/rust-lightning/pull/1219#discussion_r771853898

**Review by jkczyz — DISMISSED** [review]

**Comment by jkczyz** [comment]
Feel free to squash

**Comment by TheBlueMatt** [comment]
Squashed without diff:

```
$ git diff-tree -U1 818eb3534 7d1e1873f
$
```

**Review by jkczyz — APPROVED** [review]

**Review by arik-so — APPROVED** [review]

**Pull request merged**